Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift

```diff
@@ -24,7 +24,8 @@ class BalancesViewController: LoadableViewController, UITableViewDelegate, UITab
 
     override var isEmpty: Bool { results.isEmpty }
 
-    private var showingBanner = true
+    @UserDefault(key: "io.gnosis.multisig.importKeyBannerWasShown")
```

Comment:
We have this UserDefault in multi places of the app, are you up to create separated manager for it now same as we have for the settings and gradually refactor it?

Reply:
I am not sure what do you mean. We can discuss it and lets see if we can make it better